### ISSUE TITLE
Remove default focus style on thumbnail nav container

### ISSUE THREAD

**Issue by aeschylus** [ProjectMirador/mirador#2310]
There's a blue haze thing surrounding the thumbnail nav strip whenever you interact with it.

<img width="510" alt="Screenshot 2019-03-22 at 1 37 48 PM" src="https://user-images.githubusercontent.com/145874/54851604-badff680-4ca7-11e9-89c5-42a01c38fb77.png">

**Comment by camillevilla** [comment]
@aeschylus I think that's just the Chrome browser focus style? In firefox it's a dotted line.
![Screen Shot 2019-03-22 at 5 48 21 PM](https://user-images.githubusercontent.com/5402927/54859464-b416ab00-4cca-11e9-9559-aa7b16456ddf.png)

**Comment by camillevilla** [comment]
It seems like we still need some kind of focus style for this area if the desired behavior is: a user can focus on the thumbnail strip, use arrow controls to navigate between thumbs, and then tab to move focus to a different area of the application.

**Comment by jvine** [comment]
I don't think the container ever needs visible focus; the selected item in the container should be the thing that gets the tab focus. Only the children are interactive elements.
